lib/commands/exam.js

```diff
@@ -369,6 +378,10 @@ module.exports = TestCommand.extend({
         }
         init = true;
       }
+
+      if (typeof this.launcher !== 'undefined' && this.launcher !== null) {
+        testemEvents.recordStartedLauncherId(this.launcher.id);
+      }
```

Comment:
should this be within the `init` if block??
since `tests-start` gets called on every time a single test is started.. this path could be executed many times, and many of the times it could be duplicates

Reply:
that's very true, but if  it's within `init` it will only record the first browser that emits `tests-start` and other browsers won't be able to reach to the line to record their launch ids.

Reply:
Makes sense, feel free to resolve :)